README.md

```diff
@@ -1,13 +1,15 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project's goal is to create a RESTful API using Node in Express. The API should have endpoints which return an array of data or a single object. With the provided hardcoded dataset in JSON format with a list of over 1000 Netflix shows and show details I managed to create my first API by manipulating the data with JavaScript methods like: filter(), find() and slice().
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I first outlined the key requirements and functionality expected from the Netflix API. I identified the need for endpoints that support listing all available shows, retrieving individual shows, and handling different browsing options. Additionally, I considered potential future features, leading to the inclusion of placeholder endpoints for more complex operations.
+
+I solved it with help from my Orange teammates in the technigo bootcamp. And a bit of chatGPT for a better understanding of the code.
+
+If I had more time on my side, I would had considered to make a detailed documentation to include descriptions for each route, expected parameters, and example responses for improved usability.
```

Comment:
Maybe a list with short description/documentation here in the Read-me would be a way?